src/documents/APIChangesFrom2_0To3_0.md

```diff
@@ -12,6 +12,38 @@ This page contains a list of the features, APIs, and etc. that have changed in I
 This section describes major APIs that were in common use. For information about less widely-used features removed in
 3.0, consult [Deprecations in 2.0](DeprecatedIn2_0.md).
 
+#### DataSources
+
+A new facility for representing sources or sinks of data as Infusion components has been implemented as
+[DataSources](DataSourceAPI.md). Implementing grades derived from ```fluid.dataSource``` allows the process of
+reading and writing data, e.g. from HTTP URLs, databases, and the filesystem, to be configured as an open pipeline
+of asynchronous transformations.
+
+Note that the scheme for configuring writable variants of datasource components has changed with respect to
```

Comment:
Should this use the `infusion-docs-note` styling?

Reply:
That's okay, I was just confirming your intent. I think we can just leave it as is for now. We need to go through and clean things up and maybe create a style guide for the whole docs at some point.